nilearn/glm/first_level/first_level.py

```diff
@@ -1029,6 +1029,13 @@ def _prepare_mask(self, run_img):
         if isinstance(run_img, SurfaceImage) and not isinstance(
             self.mask_img, SurfaceMasker
         ):
+            if self.smoothing_fwhm is not None:
+                warn(
+                    "Parameter smoothing_fwhm is not "
```

Comment:
```suggestion
                    "Parameter smoothing_fwhm is not yet "
```

In case we want to support it one day?